### DISCUSSION TITLE
Unable to run openHASP on Sunton ESP32-8048S070C

### DISCUSSION THREAD

**Discussion by mak-42** [HASwitchPlate/openHASP#738] [Q&A] [answered]
I flashed "v0.7.0-rc12" from "https://nightly.openhasp.com" and it can not run. "OPENHASP" displayed during a part of asecond on the screen and the device reboots again infinitely.
The logs shows some problem on "TFT : Starting" step. 
The device itself seems to work well (I flashed lvgl demo downloaded from manufactory's site, it works).

Could you, please, prompt me, what I do wrong?

Logs:
```
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0x1 (POWERON),boot:0x28 (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x128
load:0x403c9700,len:0xb90
load:0x403cc700,len:0x28cc
entry 0x403c98b8



        open____ _____ _____ _____
          |  |  |  _  |   __|  _  |
          |     |     |__   |   __|
          |__|__|__|__|_____|__|
        Home Automation Switch Plate
        Open Hardware edition v0.7.0-rc12

[       0.082][212980/248800 14][    0/    0  0] DBUG: Started @ 115200 bps
#[       0.091][212980/248800 14][    0/    0  0] DBUG: Environment: sunton-8048s070c_16MB
#[       0.101][212980/248800 14][    0/    0  0] UART: Started
#[       0.108][212980/248800 14][    0/    0  0] UART: Client login from serial
#[       0.119][212980/248800 14][    0/    0  0] CONF: Failed to load /config.json
Prompt > [       0.129][212980/248800 14][    0/    0  0] CONF: Loading EEPROM
Prompt > [       0.138][212980/248800 14][    0/    0  0] CONF: Loaded EEPROM
Prompt > [       0.147][212980/248864 14][    0/    0  0] DBUG: Loading debug settings
Prompt > [       0.157][212980/248864 14][    0/    0  0] GPIO: Loading GUI settings
Prompt > [       0.166][212980/248864 14][    0/    0  0] HASP: Loading HASP settings
Prompt > [       0.176][212980/248864 14][    0/    0  0] WIFI: Loading WiFi settings
Prompt > [       0.202][212980/246940 13][    0/    0  0] MQTT: Loading MQTT settings
Prompt > [       0.212][212980/246940 13][    0/    0  0] TELN: Loading Telnet settings
Prompt > [       0.222][212980/246940 13][    0/    0  0] MDNS: Loading MDNS settings
Prompt > [       0.231][212980/246940 13][    0/    0  0] HTTP: Loading HTTP settings
Prompt > [       0.241][212980/246940 13][    0/    0  0] GPIO: Loading GPIO settings
Prompt > [       0.251][212980/246940 13][    0/    0  0] CONF: Settings loaded
Prompt > [       0.298][204788/215980  5][    0/    0  0] MSGR: Starting...
Prompt > [       0.307][204788/216176  5][    0/    0  0] MSGR: Started
Prompt > [       0.316][204788/216176  5][    0/    0  0] TFT : Starting...
Prompt > [       0.324][204788/216152  5][    0/    0  0] TFT : Starting...
Terminal disconnected: NetworkError: The device has been lost.
```

**Comment by TLaren** [comment]
Just about the same place, 1 line further. 
EXP32-8048D070C
The module seems good, I have several and have programmed them up for other projects using LGFX.

...
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0x1 (POWERON),boot:0x2a (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x128
load:0x403c9700,len:0xb90
load:0x403cc700,len:0x28cc
entry 0x403c98b8



        open____ _____ _____ _____
          |  |  |  _  |   __|  _  |
          |     |     |__   |   __|
          |__|__|__|__|_____|__|
        Home Automation Switch Plate
        Open Hardware edition v0.7.0-rc12

[       0.082][212980/248800 14][    0/    0  0] DBUG: Started @ 115200 bps
#[       0.091][212980/248800 14][    0/    0  0] DBUG: Environment: sunton-8048s070c_16MB
#[       0.101][212980/248800 14][    0/    0  0] UART: Started
#[       0.108][212980/248800 14][    0/    0  0] UART: Client login from serial
#[       0.119][212980/248800 14][    0/    0  0] CONF: Failed to load /config.json
Prompt > [       0.129][212980/248800 14][    0/    0  0] CONF: Loading EEPROM
Prompt > [       0.138][212980/248800 14][    0/    0  0] CONF: Loaded EEPROM
Prompt > [       0.147][212980/248864 14][    0/    0  0] DBUG: Loading debug settings
Prompt > [       0.156][212980/248864 14][    0/    0  0] GPIO: Loading GUI settings
Prompt > [       0.166][212980/248864 14][    0/    0  0] HASP: Loading HASP settings
Prompt > [       0.176][212980/248864 14][    0/    0  0] WIFI: Loading WiFi settings
Prompt > [       0.201][212980/247028 13][    0/    0  0] MQTT: Loading MQTT settings
Prompt > [       0.211][212980/247028 13][    0/    0  0] TELN: Loading Telnet settings
Prompt > [       0.221][212980/247028 13][    0/    0  0] MDNS: Loading MDNS settings
Prompt > [       0.231][212980/247028 13][    0/    0  0] HTTP: Loading HTTP settings
Prompt > [       0.240][212980/247028 13][    0/    0  0] GPIO: Loading GPIO settings
Prompt > [       0.250][212980/247028 13][    0/    0  0] CONF: Settings loaded
Prompt > [       0.297][204788/216072  5][    0/    0  0] MSGR: Starting...
Prompt > [       0.306][204788/216268  5][    0/    0  0] MSGR: Started
Prompt > [       0.314][204788/216268  5][    0/    0  0] TFT : Starting...
Prompt > [       0.323][204788/216244  5][    0/    0  0] TFT : Starting...
Prompt > [       0.354][204788/213312  3][    0/    0  0] TFT : Started
Terminal disconnected: NetworkError: The device has been lost.
...
Any help or suggestions would be appreciated

**Comment by TLaren** [comment, marked as answer]
OK, I was doing the old reset and see what changes. I touched the screen and it stopped blinking.
Is that a hint?
Then I touched again, it said to setup WiFi.
I enter my info and click the check and the logs show this.
...
[      95.664][81908/93948 12][54588/55892  3] WIFI: Connecting to instanet... 1
Prompt > [      95.924][81908/94140 12][54588/55892  3] WIFI: Connecting to instanet... 2
Prompt > [      96.184][81908/94140 12][54588/55892  3] WIFI: Connecting to instanet... 3
Prompt > [      96.444][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 4
Prompt > [      96.704][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 5
Prompt > [      96.964][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 6
Prompt > [      97.224][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 7
Prompt > [      97.484][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 8
Prompt > [      97.744][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 9
Prompt > [      98.004][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 10
Prompt > [      98.264][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 11
Prompt > [      98.524][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 12
Prompt > [      98.784][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 13
Prompt > [      99.044][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 14
Prompt > [      99.304][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 15
Prompt > [      99.564][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 16
Prompt > [      99.824][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 17
Prompt > [     100.084][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 18
Prompt > [     100.344][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 19
Prompt > [     100.604][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 20
Prompt > [     100.864][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 21
Prompt > [     101.124][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 22
Prompt > [     101.384][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 23
Prompt > [     101.644][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 24
Prompt > [     101.904][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 25
Prompt > [     102.164][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 26
Prompt > [     102.424][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 27
Prompt > [     102.684][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 28
Prompt > [     102.944][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 29
Prompt > [     103.204][81908/94144 12][54588/55892  3] WIFI: Connecting to instanet... 30
Prompt > [     103.464][81908/94096 12][54588/55892  3] WIFI: Received IP address 0.0.0.0
Prompt > [     103.474][81908/94096 12][54588/55892  3] WIFI: Received IP address 0.0.0.0
Prompt > [     156.596][81908/94420 13][55392/55900  1] MSGR: File not found: L:/idle_short.cmd
...
The screen still show the 'enter wifi Ssid and password stuff.
I then realize I have the Ssid wrong. Correct ir and I am "plate".
So I'm not sure about the flashing until I touch the screen but it seems to be up.
Just have to figure the next steps.

**Reply by mak-42** [reply]
Thanks a lot for the hint. I touched a screen, and it stopped blinking as well and I was able to setup WIFI.

**Reply by TLaren** [reply]
So what is next. 
Mine doesn't show up in HA that I can tell. Somewhere I read that it would appear in the Overview.
In Devices I see openhasp but it shows Not loaded.
I read another post about a missing yaml file but I'm still looking where to put it and what to put in it.
I did create a pages.json (actually edited it) using the web interface. Added a button, well a square for now.
using the web interface I also looked at the config.json file
...
{
    "wifi": {
        "ssid": "Instanet",
        "pass": "********"
    },
    "mqtt": {
        "host": "192.168.15.11",
        "user": "mqttu",
        "pass": "********",
        "topic": {
            "node": "hasp/%hostname%/%topic%",
            "group": "hasp/plates/%topic%",
            "broadcast": "hasp/broadcast/%topic%",
            "hass": "homeassistant/status"
        },
        "port": 1883,
        "name": "plate"
    },
    "telnet": {
        "enable": 1,
        "port": 23
    },
    "mdns": {
        "enable": 1
    },
    "http": {
        "enable": true,
        "port": 80,
        "user": "",
        "pass": "********"
    },
    "gpio": {
        "config": [
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0
        ]
    },
    "debug": {
        "ansi": 0,
        "baud": 115200,
        "tele": 300,
        "host": "",
        "port": 514,
        "proto": 0,
        "log": 0
    },
    "gui": {
        "idle1": 60,
        "idle2": 120,
        "bckl": 2,
        "bcklinv": 0,
        "rotate": 0,
        "cursor": 0,
        "invert": 0,
        "calibration": [
            0,
            65535,
            0,
            65535,
            0,
            0,
            0,
            0
        ]
    },
    "hasp": {
        "startpage": 1,
        "startdim": 255,
        "theme": 2,
        "color1": "#00b6ff",
        "color2": "#ff9962",
        "font": "",
        "pages": "/pages.jsonl"
    }
}
...
What to do next, read more I guess.

**Comment by StreakingJerry** [comment]
I have opened an bug issue for this.
[https://github.com/HASwitchPlate/openHASP/issues/754](url)
